src/js/index.js

```diff
@@ -513,16 +527,31 @@ export const hpe = deepFreeze({
     },
   },
   tab: {
-    border: {
-      color: 'text-xweak',
+    color: 'text',
+    active: {
+      background: 'background-contrast',
     },
-    color: 'text-xweak',
-    margin: {
-      horizontal: 'none',
+    hover: {
+      background: 'background-contrast',
+      color: 'text',
     },
-    pad: {
-      horizontal: 'small',
+    border: {
+      side: 'bottom',
+      color: 'border',
+      active: {
```

Comment:
On the grommet side, this should be `active.border` instead of `border.active` to align with the theme structure direction. Not an issue for this PR per-se.